### PR TITLE
Nightborne Religion

### PR DESCRIPTION
**Pull request by robmart**
## Changelog:
- Added Nightborne religion

## How to test
- Create a character with Shal'danalism religion

### CODE REVIEW

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/religion/religions/arcane_group.txt [hunk 1]
```diff
@@ -267,5 +267,38 @@
 			#Magic
 			doctrine = doctrine_light_magic_accepted
 		}
+
+		cult_of_nightwell = {
+			#Main Group
+			color = rgb { 70 54 248 }
+			#religious_head = d_cult_of_nighwell
+			#icon = wcmw_nightwell_religion
```
[the remark this answers is not in the document]
Comment (MemeWizard) [reply common/religion/religions/arcane_group.txt:281]: Fixed

File: common/religion/religions/arcane_group.txt [hunk 2]
```diff
@@ -255,17 +258,40 @@
...
+
+			#Clerical Functions
+			doctrine = doctrine_clerical_succession_temporal_fixed_appointment
+		}
+
+		cult_of_nightwell = {
+			#Main Group
+			color = rgb { 70 54 248 }
+			#religious_head = d_cult_of_nighwell
+			#icon = wcmw_nightwell_religion
```
Comment (MemeWizard) [inline common/religion/religions/arcane_group.txt:281]:
```suggestion
			icon = wcmw_nightwell_religion
```

Comment (robmart) [reply common/religion/religions/arcane_group.txt]: Thank you, you are fantastic. Let's get married.

File: common/religion/religions/arcane_group.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (robmart) [reply common/religion/religions/arcane_group.txt]: Solved in 
b3571c8

**Review by arithon — APPROVED** [review]
loc LGTM

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Review by hamjustine — APPROVED** [review]

**Pull request merged**